functions.php

```diff
@@ -2,18 +2,42 @@
 
 /**
  * Подсчёт количества задач в каждом из проектов
- * @param array $tasks список задач в виде массива
- * @param string $project_name название проекта
+ * @param array $tasks список задач
+ * @param string $project_id id проекта
  * @return int $task_quantity число задач для переданного проекта
  */
-function count_tasks(array $tasks, string $project_name) : int {
+function count_tasks(array $tasks, int $project_id) : int {
     $task_quantity = 0;
 
-    foreach ($tasks as $task) {
-        if (isset($task['project']) && $task['project'] === $project_name) {
+    foreach ( $tasks as $task ) {
+        if ((int) $task['project_id'] === $project_id) {
             ++$task_quantity;
         }
     }
 
     return $task_quantity;
 }
+
+
+/**
+ * возвращает результат запроса c плейсхолдерами
+ * @param mysqli $connect параметры соединения
+ * @param string $sql_query sql-запрос
+ * @param  data //
+ */
+function get_sql_result($connect, $sql_query, $data) {
+    $stmt = mysqli_stmt_init($connect);
+
+    if(!mysqli_stmt_prepare($stmt, $sql_query)) {
+        $errorMsg = 'Не удалось инициализировать подготовленное выражение: ' . mysqli_error($connect);
+        die($errorMsg);
+
+    } else {
+        mysqli_stmt_bind_param($stmt, "i", $data);
+        mysqli_stmt_execute($stmt);
```

Comment:
Для того чтобы твоя функция могла делать запросы с более чем 1 параметром, нужно в $data передать массив (как в db_get_prepare_stmt).
Я так понимаю warning при работе с ней у тебя был warning т.к. вызов был с передачей не массива, а просто $current_user_id.

```suggestion
		if ($data) {
			mysqli_stmt_bind_param($stmt, "s", ...(array)$data);
		}
        mysqli_stmt_execute($stmt);
```

Пишу код на гитхабе, запускать не пробовал. Надеюсь ошибки тут нет